ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/424_model_serving_bias_metrics.robot

```diff
@@ -0,0 +1,151 @@
+${PRJ_TITLE}=                        model-serving-project
+${MODEL_ALPHA}=                      demo-loan-nn-onnx-alpha
+${MODEL_PATH_ALPHA}=                 trusty/loan_model_alpha.onnx
+${MODEL_BETA}=                       demo-loan-nn-onnx-beta
+${MODEL_PATH_BETA}=                  trusty/loan_model_beta.onnx
+${TRUSTYAI_RESOURCEPATH}=            ods_ci/tests/Resources/Files/TrustyAI
+${TRUSTYAI_CR_FILEPATH}=             ${TRUSTYAI_RESOURCEPATH}/trustyai_crd.yaml
+${MONITORING_CONFIG_FILEPATH}=       ods_ci/tests/Resources/Files/uwm_cm_conf.yaml
+${UWM_CONFIG_FILEPATH}=              ods_ci/tests/Resources/Files/uwm_cm_enable.yaml
+${aws_bucket}=                       rhods-public
+${RUNTIME_NAME}=                     Model Bias Serving Test
+${PRJ_DESCRIPTION}=                  Model Bias Project Description
+${framework_onnx}=                   onnx - 1
+
+*** Test Cases ***
+Verify Bias Metrics Available In CLI For Models Deployed Prior To Enabling Trusty Service For Admin User
+    [Documentation]    Verifies that the Bias metrics are available in the Prometheus for a model which was
+    ...                 deployed prior to enabling the TrustyAI service
+    [Tags]    Smoke
+    ...       Tier1   ODS-2482    ODS-2479
+    Open Model Serving Home Page
+    Serve Model    project_name=${PRJ_TITLE}    model_name=${MODEL_ALPHA}    framework=${framework_onnx}    existing_data_connection=${TRUE}
+    ...    data_connection_name=model-serving-connection    model_path=${MODEL_PATH_ALPHA}    model_server=${RUNTIME_NAME}
+    ${runtime_name} =    Replace String Using Regexp    string=${RUNTIME_NAME}    pattern=\\s    replace_with=-
+    ${runtime_name} =    Convert To Lower Case    ${runtime_name}
+    Wait For Pods To Be Ready    label_selector=name=modelmesh-serving-${runtime_name}    namespace=${PRJ_TITLE}
+    Verify Model Status    ${MODEL_ALPHA}    success
+    Install And Verify TrustyAI Service
+    Wait Until Keyword Succeeds  5 min  10 sec  Verify Model Is Registered with TrustyAI Service
+    Send Batch Inference Data to Model     model_name=${MODEL_ALPHA}     project_name=${PRJ_TITLE}
+    ${modelId}=   Get ModelId For A Deployed Model    modelId=${MODEL_ALPHA}
+    ${modelId}    Replace String    ${modelId}    "    ${EMPTY}
+    Schedule Bias Metrics request via CLI     metricsType=dir   modelId=${modelId}    protectedAttribute="customer_data_input-3"
+    ...       favorableOutcome=0   outcomeName="predict"    privilegedAttribute=1.0    unprivilegedAttribute=0.0
+    Verify TrustyAI Metrics Exists In Observe Metrics    trustyai_dir    retry_attempts=2    user_type=basic
+    Remove TrustyAI Service
+
+*** Keywords ***
+Bias Metrics Suite Setup
+    [Documentation]    Setup to configure TrustyAI metrics
+    Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
+    Log    Enabling and Configuring User Workload Monitoring
+    Enable User Workload Monitoring
+    Configure User Workload Monitoring
```

Comment:
could you invert the order? check the second NOTE in the section [Enabling monitoring for user-defined projects](https://docs.openshift.com/container-platform/4.14/monitoring/enabling-monitoring-for-user-defined-projects.html)